assets/user-profile/actions.ts

```diff
@@ -340,3 +333,19 @@ export function updateUserNotificationSchedules(schedule: Omit<IUser['notificati
             .catch((error) => errorHandler(error, dispatch, setError(error)));
     };
 }
+
+export function setTopicSubscribers(topic: ITopic, subscribers: ITopic['subscribers']) {
+    return (dispatch: any, getState: any) => {
+        const updates = {subscribers};
+        return updateTopic(topic, updates, dispatch);
+    };
+}
+
+function updateTopic(topic: ITopic, updates: any, dispatch: any) {
```

Comment:
Types can be improved here to enforce correct updates are supplied, `updates: Partial<ITopic>` and return type `Promise<void>`

Reply:
added updates type, seems like return type it gets right automatically